scripts/common/addon.sh

```diff
@@ -105,6 +110,17 @@ function addon_load() {
     load_images $DIR/addons/$name/$version/images
 }
 
+function addon_fetch() {
+    local url=$1
+
+    local archiveName=$(basename $url)
```

Comment:
Do we prefer snake or camel case? Seems to be a mix in this pr

Reply:
Hmm. Looking at the other scripts, locals are camel case and functions are snake case. I'll change my PR to be consistent. IMHO I would make everything snake case - looks like that's the [Google Style guide ](https://google.github.io/styleguide/shellguide.html#s7-naming-conventions) as well.